core/indigo-core/graph/graph.h

```diff
@@ -253,6 +268,8 @@ namespace indigo
         bool _components_valid;
         int _components_count;
 
+        PtrArray<GraphMetaObject> _meta_data;
```

Comment:
Why do we use PtrArray? It should be considered as deprecated data structure now.

Reply:
We can't use std containers here